File: Inventarization.Api/Inventorization.Business/Model/Action.cs
using System;

namespace Inventorization.Business.Model
{
    public class Action
    {
        public Guid Id { get; set; }
        public DateTime DateTime { get; set; }
        public ActionType Type { get; set; }
        public Guid UserId { get; set; }
        public string BarCode { get; set; }
        public Guid Zone { get; set; }
        public Guid Inventorization { get; set; }
        public int Quantity { get; set; }
    }
}

File: Inventarization.Android.Client/app/src/main/java/inventorization/Action.java
package inventorization;

import java.util.Date;

/**
 * Created by Барашики on 04.04.2017.
 */


public class Action {
    public String Id;
    public Date DateTime;
    public ActionType Type;
    public String UserId;
    public String BarCode;
    public String Zone;
    public String Inventorization;
    public Integer Quantity;
    public ActionStatus Status;
}
